Azure/azure-cli
# [AppService] Fixes #14583: az webapp up should generate default name if name isn't provided

**Description**
- Make `--name` parameter optional for command `az webapp up`
- If name isn't provided, generate default name in the format adjective-noun-guid, ex. zealous-tree-06c552e1e

- Also fixes #16185 documentation update

**Testing Guide**
- run `az webapp up` without parameters in a directory you want to deploy
- a default name should be generated for you

![image](https://user-images.githubusercontent.com/14339314/102141611-f0caf400-3e15-11eb-87e4-214064c76cfb.png)

---

This checklist is used to make sure that common guidelines for a pull request are followed.
